spec/models/set_member_subject_spec.rb

```diff
@@ -152,15 +152,15 @@
         let(:subject_ids) { [] }
 
         it "should return the all the worflow set_member_subjects" do
-          expect(SetMemberSubject.unseen_for_user_by_workflow(user, workflow)).to match_array(smses)
```

Comment:
method expects a user Id and workflow Id. when sending over just the object, the query that expects something along the lines of `WHERE user.id=user_id`,comes up `WHERE user.id=NULL`. 

I believe it has something to do along the lines with https://github.com/rails/arel/issues/531, or something similar. 
(It seems that this may be broken until we get to rails 5.2 [where we send an object when an id is expected)

POSSIBLE TODO (in a followup PR) -
update https://github.com/zooniverse/panoptes/blob/b9e1e605285e01c15d9e2e45b1515a190989a11e/lib/subjects/set_member_subject_selector.rb#L26-L31 reference to send over `user.id` and `workflow.id`